packages/SwingSet/src/kernel/virtualObjectManager.js

```diff
@@ -158,8 +160,9 @@ export function makeCache(size, fetch, store) {
 export function makeVirtualObjectManager(
   syscall,
   allocateExportID,
-  valToSlot,
-  registerValue,
+  getSlotForVal,
+  getValForSlot,
```

Comment:
This isn't used so (a) why is it here and (b) why isn't eslint whining about its unusedness?

Reply:
It gets used in the next PR in the series.

I have no idea why eslint didn't complain. Maybe it's looking into the future. Not gonna complain about it not complaining :)